src/main/webapp/script.js

```diff
@@ -1,23 +1,23 @@
 function initMap() {
-  var foodLocations = { 'mozerella': {lat: 1.326574, lng: 103.814158},
-                        'pastries': {lat: 1.322040, lng: 103.814554},
-                        'curry puffs': {lat: 1.323561, lng: 103.841931} };
+  var foodLocations = [{'position': {lat: 1.326574, lng: 103.814158}, 'desc': 'Mozerella cheeseball' },
```

Comment:
Minor typo. The correct spelling is mozzarella! Go ahead and change this typo before merging.

Reply:
Also it might make more sense to store the Name and description of the shop but maybe we can do that later